presto-main/src/main/java/io/prestosql/operator/DynamicFilterSourceOperator.java

```diff
@@ -129,34 +137,46 @@ public OperatorFactory duplicate()
     private final Consumer<TupleDomain<DynamicFilterId>> dynamicPredicateConsumer;
     private final int maxFilterPositionsCount;
     private final long maxFilterSizeInBytes;
+    private final boolean minMaxValues;
 
     private final List<Channel> channels;
+    private final List<Integer> orderableChannels;
 
     // May be dropped if the predicate becomes too large.
     @Nullable
     private BlockBuilder[] blockBuilders;
     @Nullable
     private TypedSet[] valueSets;
+    @Nullable
+    private Object[] minValues;
```

Comment:
let's just keep it as `Block[]` that contain singletons

Reply:
done